data/hai/hai outfits.txt

```diff
@@ -812,6 +812,27 @@ outfit "Pulse Rifle"
 	"unplunderable" 1
 	description "The Pulse Rifle was first developed by the Hai to help protect their ships from boarders during the Korath raids. While it has since been adapted to more effectively incapacitate its target, if several bolts land in quick succession, the accumulated charge has a lethal effect on most species' nervous systems."
 
+outfit "Dual Pulse Pistols"
+	category "Hand to Hand"
+	plural "Dual Pulse Pistols"
+	cost 77777
+	thumbnail "outfit/dual pulse pistols"
+	"capture attack" 2.6
+	"capture defense" 1.6
+	"unplunderable" 1
+	description "A pair of modified heavy pulse pistols that are favored by some Unfettered and the occasional pirate captain. Dual wielding them adds more firepower during short raids against a ship's defenders. Individually, they lose their energy charge more quickly than a pulse rifle, making them a poor choice for holding a defensive position during a long siege."
+
+outfit "Heshi"
+	category "Unique"
+	cost 10000
+	thumbnail "outfit/unknown"
+	"mass" 1
+	"capture attack" 4
+	"capture defense" 8
+	"maintenance costs" 100
+	"unplunderable" 1
+	description "This is an aquarium containing what the Hai call a 'Heshi' - a huge scorpion looking creature. It must be fed and cared for but can be very useful in damaging the morale of enemies attacking the ship."
+	description "	On the offensive it can be used to intimidate enemies into surrendering, when they are made aware that they could become its next meal if they do not cooperate... Furthermore, it is very useful in the event of an interrogation."
```

Comment:
```suggestion
	description "	On the offensive it can be used to intimidate enemies into surrendering, when they are made aware that they could become its next meal if they do not cooperate... furthermore, it is very useful in the event of an interrogation."
```